### PR TITLE
{bio}[system/system] MAGeCK-VISPR v0.5.5 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by jordiblasco**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/69657ba7e905aac6e2783bbec7d3fba1 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node366.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/575b7d60f0ea53f8b425a650c90d0328 for a full test report.

File: easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.5-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+source_urls = ['https://bitbucket.org/liulab/%(namelower)s/get']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7e6867c68316ed35f059127cca8f78e6a0a609c274f9448579575449b11095e2']
+
+dependencies = [
+    ('Miniconda3', '4.7.10', '', True),
+    ('Python', '3.7.4', '-GCCcore-8.3.0')
```
Comment (lexming) [inline easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.5-Python-3.7.4.eb:25]: Having MAGeCK-VISPR on `SYSTEM` and this dependency on Python will cause issues. If there is any other module of Python already loaded, loading MAGeCK-VISPR will fail. The solution might be as easy as removing the dependency on Python though, as it is already installed by conda in any case
```
[...]
    python-3.7.8               |h6f2ec95_1_cpython        53.1 MB  conda-forge
    python-dateutil-2.8.1      |             py_0         220 KB  conda-forge
    python-irodsclient-0.8.2   |             py_0          69 KB  conda-forge
[...]
```

**Review by lexming — CHANGES_REQUESTED** [review]
Thanks for this PR, the installation works well. However, there is an issue with the dependency on Python. Please check my comment below

**Comment by verdurin** [comment]
@jordiblasco any thoughts on the comment from @lexming ?

**Review by jfgrimm — COMMENTED** [review]
As decided in issue #16330, we have deprecated the use of `True` to signify a system-toolchain dependency (#16384), in favour of the more intuitive `SYSTEM` template constant. Due to the change in the test suite, please run `eb --sync-pr-with-develop 11560` and update the PR to use `SYSTEM` instead.

**Comment by jfgrimm** [comment]
closing since this has gone stale, and the review comments have gone unaddressed

**Pull request closed without merging**